Modules/_cursesmodule.c

```diff
@@ -3071,8 +3071,6 @@ _curses_init_pair_impl(PyObject *module, short pair_number, short fg,
     return PyCursesCheckERR(init_pair(pair_number, fg, bg), "init_pair");
 }
 
-static PyObject *ModDict;
```

Comment:
Ah! That's awful! Thanks for removing it.

Reply:
I should note that every Cython module out there does this for performance reasons. That's one of the reasons that makes it difficult to convince myself to switch to full-blown PEP-489 support – the necessary lookup indirections are just too much of a drawback.